packages/storybook/src/generators/configuration/configuration-nested.spec.ts

```diff
@@ -105,6 +105,20 @@ describe('@nrwl/storybook:configuration for workspaces with Root project', () =>
       expect(tree.exists('.storybook/main.ts')).toBeTruthy();
       expect(tree.exists('.storybook/tsconfig.json')).toBeTruthy();
       expect(tree.exists('.storybook/preview.ts')).toBeTruthy();
+
+      expect(tree.read('.storybook/main.ts', 'utf-8')).toMatchSnapshot();
```

Comment:
Since we're checking snapshots, the previous existence assertions aren't needed.